### PR TITLE
Documentation spelling corrections

### PR DESCRIPTION
**Pull request by MallocArray**

### CODE REVIEW

File: build/startup/index.md [hunk 1]
```diff
@@ -160,11 +160,11 @@ Before the 0,0 point and Z endstop locations are set, the physical locations of
 
 ### V1, Legacy:
 
-The Z endstop should be located at close to max X position.  Home X and Y with `G28 X Y`  and then traverse just Y to locate a Z endstop position at the maximum X travel that will still trigger the endstop.  Lock down the Z endstop at that position.
+The Z endstop should be located close to max X position.  Home X and Y with `G28 X Y`  and then traverse just Y to locate a Z endstop position at the maximum X travel that will still trigger the endstop.  Lock down the Z endstop at that position.
```
Comment (nemgrea) [inline build/startup/index.md:163]: this should say **as** close to max as possible

File: build/startup/index.md [hunk 2]
```diff
@@ -160,11 +160,11 @@ Before the 0,0 point and Z endstop locations are set, the physical locations of
...
 
 ### Trident:
 
-The Z endstop should be located at close to max Y position.  Home X and Y with `G28 X Y`  and then traverse just X to locate a Z endstop position at the maximum Y travel that will still trigger the endstop.  Lock down the Z endstop at that position.
+The Z endstop should be located close to max Y position.  Home X and Y with `G28 X Y`  and then traverse just X to locate a Z endstop position at the maximum Y travel that will still trigger the endstop.  Lock down the Z endstop at that position.
```
Comment (nemgrea) [inline build/startup/index.md:167]: same as above missing the word **as**

**Review by nemgrea — CHANGES_REQUESTED** [review]
see comments on max x position edit

**Comment by MallocArray** [comment]
@nemgrea 
Addressed comment

File: build/startup/index.md (hunk 1 above)
Comment (MallocArray) [reply build/startup/index.md]: Addressed

File: build/startup/index.md (hunk 2 above)
Comment (MallocArray) [reply build/startup/index.md]: Addressed

**Review by nemgrea — APPROVED** [review]

**Pull request merged**